src/test/java/seedu/address/storage/JsonAdaptedFriendTest.java

```diff
@@ -17,7 +17,7 @@
 
 // TODO: Update tests for FriendId and Games based on UG specs
```

Comment:
I know this wasn't written by you, but consider removing it as the tests have been updated.